src/api/libcellml/analysermodel.h

```diff
@@ -461,6 +461,26 @@ class LIBCELLML_EXPORT AnalyserModel
      */
     bool needAcothFunction() const;
 
+    /**
+     * @brief Test to determine if @p variable1 and @p variable2 are the same or
+     * (directly or indirectly) equivalent.
+     *
+     * Test to see if @p variable1 is the same or (directly or indirectly)
```

Comment:
no need for all the (directly or indirectly) - equivalence is equivalence.

Reply:
Directly or indirectly doesn't refer to directionality. IOW, in `A <-> B <-> C`, `A` is directly equivalent to `B` and indirectly equivalent to `C`.

Reply:
nothing to do with directionality - simply trying to say that direct or indirect equivalence makes no difference here so no need to keep mentioning it in the documentation.

Reply:
Argh, fair point. Fixed in commit e168fc038.